### ISSUE TITLE
Have an "Activity Rating"

### ISSUE THREAD

**Issue by nathanmerrill** [vihanb/PPCG-Design#59]
I'd be cool if there was an "Activity rating" calculated by:

(Number actions performed)/(Total time spent on site) 

The statistic would persist from session to session, and perhaps different activities would increase it by different amounts: (Posting a question: 20, Posting an answer: 10, Voting: 1, Commenting: 2)

Note that this is super optional, but thought that I might share my idea. :)

**Comment by vihanb** [comment]
http://data.stackexchange.com/codegolf/query/441791/activity-score?UserId=40695

**Comment by The-Quill** [comment]
@vihanb It'd need to be an API/JS thing not a SEDE thing, then

**Comment by vihanb** [comment]
@The-Quill  yes I just realized that and now I am very sad right now that I wasted like 2 hours making something I can't even use :(

**Comment by The-Quill** [comment]
Another issue with that is that you'll probably end up loading the Activity rating for many pages, so you'll effectively be calling the API quite a bit. You may use a lot of API reqs doing that. Just something to consider.

**Comment by huesatlum** [comment]
Yeah, it's not the type of thing you need to see on every page. If you do it, make it a button or something that doesn't make any requests until clicked.

**Comment by nathanmerrill** [comment]
I think it'd be best to store it client side.  If you wanted it to persist, you could make an api call to synchronize every day or so, but making another api call every page load is a bit much.
